rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -2087,15 +2094,15 @@ static NEVER_INLINE error_code savedata_get_list_item(vm::cptr<char> dirName, vm
 
 		for (const auto& entry : fs::dir(save_path))
 		{
-			if (entry.is_directory)
+			if (entry.is_directory || check_filename(vfs::unescape(entry.name), false, false))
 			{
 				continue;
 			}
 
 			size_kbytes += ::narrow<u32>((entry.size + 1023) / 1024); // firmware rounds this value up
 		}
 
-		*sizeKB = size_kbytes;
+		*sizeKB = size_kbytes + 35;
```

Comment:
nani?

Reply:
https://github.com/RPCS3/rpcs3/blob/266b605f45f115f79f6d55a85a0cc227ae5dc721/rpcs3/Emu/Cell/Modules/cellSaveData.cpp#L1510

Reply:
So why not add a comment or whatever

Reply:
Done